### ISSUE TITLE
validation error

### ISSUE THREAD

**Issue by LiveAndLearn322** [wyf0912/LLFlow#9]
I trained this model in LOL, the validation PSNR and SSIM are normal at first, but then dropped suddenly. I only modified the batch size=8, why does this happen?
![Snipaste_2022-04-14_10-32-54](https://user-images.githubusercontent.com/74809841/163303568-646c2afb-a956-4861-a77c-7feef5e3bfc9.png)

**Comment by wyf0912** [comment]
If you could not use the same batch size, you can try to decrease the learning rate.

This may be related to the stability of the normalizing flow.

**Comment by wyf0912** [comment]
You could reopen the issue if you meet the problem again
